vou/drug_params.json

```diff
@@ -0,0 +1,128 @@
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "hydrocodone":{
+            "Injection":0.1,
+            "Insufflation":0.1,
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "hydromorphone":{
+            "Injection":0.1,
+            "Insufflation":0.1,
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "levorphanol_tartrate":{
+            "Injection":0.1,
+            "Insufflation":0.1,
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "meperidine_hcl":{
+            "Injection":0.1,
+            "Insufflation":0.1,
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "morphine":{
+            "Injection":0.15,
+            "Insufflation":0.15,
+            "Smoking":0.15,
+            "Pill":0.55
+        },
+        "oxycodone":{
+            "Injection":0.15,
+            "Insufflation":0.15,
+            "Smoking":0.15,
+            "Pill":0.55
+        },
+        "oxymorphone":{
+            "Injection":0.15,
+            "Insufflation":0.15,
+            "Smoking":0.15,
+            "Pill":0.55
+        },
+        "pentazocine":{
+            "Injection":0.1,
+            "Insufflation":0.1,
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "tapentadol":{
+            "Injection":0.1,
+            "Insufflation":0.1,
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "tramadol":{
+            "Injection":0.1,
+            "Insufflation":0.1,
+            "Smoking":0.1,
+            "Pill":0.7
+        },
+        "heroin":{
+            "Injection":0.5,
+            "Insufflation":0.2,
+            "Smoking":0.2,
+            "Pill":0.1
+        },
+        "fentanyl":{
+            "Injection":0.2,
+            "Insufflation":0.2,
+            "Smoking":0.3,
+            "Pill":0.2
+        }
+    },
+    "drugs_by_source":{
+        "secondary_doctor":{
+            "codeine": 0.083,
+            "dihydrocodeine": 0.083,
+            "hydrocodone": 0.083,
+            "hydromorphone": 0.083,
+            "levorphanol_tartrate": 0.083,
+            "meperidine_hcl": 0.083,
+            "morphine": 0.083,
+            "oxycodone": 0.083,
+            "oxymorphone": 0.083,
+            "pentazocine": 0.083,
+            "tapentadol": 0.083,
+            "tramadol": 0.083,
+            "heroin":0.0,
+            "fentanyl":0.0
+        },
+        "dealer":{
+            "codeine": 0.1,
+            "dihydrocodeine": 0.1,
+            "hydrocodone": 0.1,
+            "hydromorphone": 0.1,
+            "levorphanol_tartrate": 0.0,
+            "meperidine_hcl": 0.0,
+            "morphine": 0.1,
+            "oxycodone": 0.1,
+            "oxymorphone": 0.1,
+            "pentazocine": 0.0,
+            "tapentadol": 0.0,
+            "tramadol": 0.0,
+            "heroin":0.2,
+            "fentanyl":0.1
+        }
+    },
+    "source_probability":{
```

Comment:
Also don't know why this is showing up as a merge conflict rather than an addition...

Reply:
These are still showing up as merge conflicts. Did you try merging `staging` into feature?